### PR TITLE
Add senta dy2static test for ce platform

### PR DESCRIPTION
**Pull request by Ray2020BD**
Add senta dy2static test for ce platform

### CODE REVIEW

File: dy2static/unittest/sentiment_dy2static/getdata.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        def reader():
+            for epoch_index in range(epoch):
+                for doc, label, seq_len in all_data:
+                    yield doc, label, seq_len
+
+        return reader
+
+    #def get_train_examples():
+    #    return data_reader ()
```
Comment (Aurelius84) [inline dy2static/unittest/sentiment_dy2static/getdata.py]: please remove the commented codes~

File: dy2static/unittest/sentiment_dy2static/model.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+                cost_time += (time.time() - start_time) * 1000  # ms
+
+                total_loss += avg_cost
+                total_acc += acc
+                total_sample += 1
+
+                if (batch_id + 1) % args.log_step == 0:
+                    #time_end = time.time()
+                    #used_time = time_end - time_begin
```
Comment (Aurelius84) [inline dy2static/unittest/sentiment_dy2static/model.py]: please clean the commented codes~~

**Review by Aurelius84 — COMMENTED** [review]
Gread~~, LGTM for the rest

File: dy2static/unittest/sentiment_dy2static/getdata.py (hunk 1 above)
Comment (Ray2020BD) [reply dy2static/unittest/sentiment_dy2static/getdata.py]: Done.

File: dy2static/unittest/sentiment_dy2static/model.py (hunk 2 above)
Comment (Ray2020BD) [reply dy2static/unittest/sentiment_dy2static/model.py]: Done.

**Review by Aurelius84 — APPROVED** [review]

**Pull request merged**